bridge/README.md

```diff
@@ -120,10 +132,118 @@ kubectl delete -f deploy/service.yaml
    set API_URL=http://keptn.127.0.0.1.nip.io/api
    set API_TOKEN=1234-exam-ple
    ```
-1. Run `npm run start:dev` from bridge root level to start the express server and the Angular app.
-1. Access the web through the url shown on the console (e.g., http://localhost:3000/ ).
+4. Run `npm run start:dev` from bridge root level to start the express server and the Angular app.
+5. Access the web through the url shown on the console (e.g., http://localhost:3000/ ).
+
+## IDE Setup
+
+### Create workspace
+
+Before creating the workspace, make sure that you already cloned the keptn/keptn repository to your local file system.<br/>
```

Comment:
The `<br/>` is not necessary here I guess?

Reply:
I would leave it as it is - as it is additional information for IntelliJ and is then more readable / visible :)

Reply:
Removed it 😆